### ISSUE TITLE
Installation of custom PhantomJS fails when it's not available in npm

### ISSUE THREAD

**Issue by luisg123v** [Vauxoo/maintainer-quality-tools#273]
We are currently using a custom source to install  PhantomJS, other tan npm (Bitbucket). However, NPM installation is still performed [here](https://github.com/Vauxoo/maintainer-quality-tools/blob/e1c64116ed7771232a780f414afc7274395a8858/travis/travis_install_nightly#L132) after Bitbucket installation [here](https://github.com/Vauxoo/maintainer-quality-tools/blob/e1c64116ed7771232a780f414afc7274395a8858/travis/travis_install_nightly#L57).

If we try to install the latest version of PhantomJS (2.1.1) npm installation fails:

```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for phantomjs-prebuilt@2.1.1
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2018-06-28T21_31_29_061Z-debug.log
```

After that,  the installation via Bitbucket fails:

```
Installing phantomjs-2.1.1
cp: not writing through dangling symlink ‘/root/maintainer-quality-tools/travis/phantomjs’
1.9.0
```

This makes me wonder:
1. Is the npm installation really required?
1. Why version 2.1.1 is not available in npm anymore? Since PhantomJS development has been abandoned (https://github.com/ariya/phantomjs/issues/15344) latest version in all sources should be 2.1.1

**Comment by luisg123v** [comment]
CC @moylop260  & @yaniaular

**Comment by moylop260** [comment]
FYI using OCA way the version 2.1.1 does not exist.

You can test it using:
`docker run -it --rm  vauxoo/odoo-80-image-shippable-auto bash`

```bash
npm install --prefix / phantomjs-prebuilt@2.1.1
npm ERR! code ETARGET
npm ERR! notarget No matching version found for phantomjs-prebuilt@2.1.1
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2018-06-29T17_56_34_316Z-debug.log
```

`npm install --prefix / phantomjs-prebuilt@2.1` install as well.

See more details:
 - ![Screen_Shot_2018-06-29_at_12.57.32](/uploads/3e768cedc84ad626a6d6215a331cf105/Screen_Shot_2018-06-29_at_12.57.32.png)

You need change
```diff
- PHANTOMJS_VERSION: "2.1.1"
+ PHANTOMJS_VERSION: "2.1"
```

Or use another valid subversion:
```diff
- PHANTOMJS_VERSION: "2.1.1"
+ PHANTOMJS_VERSION: "2.1.2"
```

**Comment by luisg123v** [comment]
@moylop260 
Yes, it stop working yesterday, and that triggered this issue. Before that, it was installing successfully, so we didn't notice anything.

BTW, Thanks for fixing this.

**Comment by luisg123v** [comment]
mmm, now I got it, I though npm installation had suddenly stop working, I hadnt seen https://github.com/Vauxoo/maintainer-quality-tools/pull/272.
